### ISSUE TITLE
Information about Full supervision model from the paper

### ISSUE THREAD

**Issue by aniket-gupta1** [nv-dvl/segment-anything-lidar#6]
Hi, Thanks for your amazing work. Looking forward for the code release. 

I had a doubt from the paper - In table 5, for the SAL (Full) model, do you take the class predictions from the CLIP branch or do you train a softmax classifier? 

If I understand correctly, SAL has learnable queries for identifying object instances, for each query, you predict a CLIP token which identifies the class of that object. So, by full supervision model, do you mean that you train the model with full supervision on each instance from the ground truth labels or do you train it using a softmax classifier like other instance segmentation approaches?
